bigquery/samples/tests/test_dataset_samples.py

```diff
@@ -20,3 +20,4 @@ def test_create_dataset(capsys, client, random_dataset_id):
     create_dataset.create_dataset(client, random_dataset_id)
     out, err = capsys.readouterr()
     assert "Created dataset {}".format(random_dataset_id) in out
+    client.delete_dataset(random_dataset_id, delete_contents=True, not_found_ok=True)
```

Comment:
Not needed here. The fixture should take care of it for you. It's better to use fixtures for this because if the test fails, the clean-up code in the fixture will still run.

Reply:
Ignore. I see you reverted this changed.